/**:
  ros__parameters:
    avoidance_by_lane_change:
      execute_object_longitudinal_margin: 80.0
      execute_only_when_lane_change_finish_before_object: false

      # avoidance is performed for the object type with true
      target_object:
        car:
          execute_num: 2                               # [-]
          moving_speed_threshold: 1.0                  # [m/s]
          moving_time_threshold: 1.0                   # [s]
          max_expand_ratio: 0.0                        # [-]
          envelope_buffer_margin: 0.3                  # [m]
          avoid_margin_lateral: 1.0                    # [m]
          safety_buffer_lateral: 0.7                   # [m]
        truck:
          execute_num: 2
          moving_speed_threshold: 1.0                  # 3.6km/h
          moving_time_threshold: 1.0
          max_expand_ratio: 0.0
          envelope_buffer_margin: 0.3
          avoid_margin_lateral: 1.0
          safety_buffer_lateral: 0.7
        bus:
          execute_num: 2
          moving_speed_threshold: 1.0                  # 3.6km/h
          moving_time_threshold: 1.0
          max_expand_ratio: 0.0
          envelope_buffer_margin: 0.3
          avoid_margin_lateral: 1.0
          safety_buffer_lateral: 0.7
        trailer:
          execute_num: 2
          moving_speed_threshold: 1.0                  # 3.6km/h
          moving_time_threshold: 1.0
          max_expand_ratio: 0.0
          envelope_buffer_margin: 0.3
          avoid_margin_lateral: 1.0
          safety_buffer_lateral: 0.7
        unknown:
          execute_num: 1
          moving_speed_threshold: 0.28                 # 1.0km/h
          moving_time_threshold: 1.0
          max_expand_ratio: 0.0
          envelope_buffer_margin: 0.3
          avoid_margin_lateral: 1.0
          safety_buffer_lateral: 0.0
        bicycle:
          execute_num: 2
          moving_speed_threshold: 0.28                 # 1.0km/h
          moving_time_threshold: 1.0
          max_expand_ratio: 0.0
          envelope_buffer_margin: 0.8
          avoid_margin_lateral: 1.0
          safety_buffer_lateral: 1.0
        motorcycle:
          execute_num: 2
          moving_speed_threshold: 1.0                  # 3.6km/h
          moving_time_threshold: 1.0
          max_expand_ratio: 0.0
          envelope_buffer_margin: 0.8
          avoid_margin_lateral: 1.0
          safety_buffer_lateral: 1.0
        pedestrian:
          execute_num: 2
          moving_speed_threshold: 0.28                 # 1.0km/h
          moving_time_threshold: 1.0
          max_expand_ratio: 0.0
          envelope_buffer_margin: 0.8
          avoid_margin_lateral: 1.0
          safety_buffer_lateral: 1.0
        lower_distance_for_polygon_expansion: 0.0      # [m]
        upper_distance_for_polygon_expansion: 1.0      # [m]

      # For target object filtering
      target_filtering:
        # avoidance target type
        target_type:
          car: true                                    # [-]
          truck: true                                  # [-]
          bus: true                                    # [-]
          trailer: true                                # [-]
          unknown: true                                # [-]
          bicycle: false                               # [-]
          motorcycle: false                            # [-]
          pedestrian: false                            # [-]

      constraints:
        # lateral constraints
        lateral:
          velocity: [1.0, 1.38, 11.1]                   # [m/s]
          max_accel_values: [0.5, 0.5, 0.5]             # [m/ss]
          min_jerk_values: [0.2, 0.2, 0.2]              # [m/sss]
          max_jerk_values: [1.0, 1.0, 1.0]              # [m/sss]
